views/js/runner/plugins/security/blurPause.js

```diff
@@ -120,32 +121,44 @@ define([
                 // select element on iOS devices for some reason lost focus when an option is selected
                 // but after some delay the focus is restored back to the element
                 _.delay(function(){
-                    if(!mainFocus && !innerFocus){
+                    if(!mainFocus && !innerFocus && !ckEditorFocus){
                         //the inner window has lost the focus and no one else has it
                         doPause();
                     }
                 }, focusBackTimeoutDelayMs);
             };
 
+            var handleCkEditorFocus = function handleCkEditorFocus(){
```

Comment:
Such function to handle a boolean is a little bit overkill, you might consider merging the two handlers or just simply remove them and pass the boolean and the doPause as callback to the event handler.

Reply:
The second variant will not work because `ckEditorFocus` and `innerFocus` handled separately